AD419.DataHelper.Web/Helpers/ProcessCategoryExtensions.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Data.Entity;
+using System.Linq;
+using AD419.DataHelper.Web.Models;
+
+namespace AD419.DataHelper.Web.Helpers
+{
+    public static class ProcessCategoryExtensions
+    {
+        public static ProcessCategory GetProcessCategory(this AD419DataContext context, ProcessStatuses status)
+        {
+            // returns the process status for the given ID
+            var processStatus = ProcessStatusExtensions.GetProcessStatus(context, status);
+
+            var processCategory = context.ProcessCategories.FirstOrDefault(c => c.Id == processStatus.CategoryId);
+
+            if (processCategory == null)
+            {
+                throw new ArgumentException(string.Format("Could not find process category for process status {0}",
+                    status));
+            }
+
+            return processCategory;
+        }
+
+        public static ProcessCategory GetNextProcessCategory(this AD419DataContext context, ProcessStatuses status)
+        {
+            var processCategory = GetProcessCategory(context, status);
+
+            var nextProcessCategory = context.ProcessCategories.Where(c => c.SequenceOrder == processCategory.SequenceOrder + 1).

```

Comment:
@taylorkj do you need the .Include() most of the time?  It seems like as part of usage you are only getting it to determine the process category so you can set it completed, and you aren't using the Statuses associated property.  If you do want it every time then this is fine, but if not it's doing extra data fetching